lib/Dialect/TritonGPU/Transforms/CoalesceAsyncCopy.cpp

```diff
@@ -0,0 +1,125 @@
+#include "mlir/Support/LLVM.h"
+#include "mlir/Transforms/Passes.h"
+#include "triton/Analysis/Utility.h"
+#include "triton/Conversion/TritonGPUToLLVM/Utility.h"
```

Comment:
nit: this is a bit of a layering violation, getRegToSharedLayout probably belongs to triton gpu dialect utils.

Reply:
can you remove the include now?

Reply:
oh right I forgot - just removed